pom.xml

```diff
@@ -178,30 +176,19 @@
             </property>
          </activation>
       </profile>
+      <!-- At the moment we only have on Weld profile - keep this for future versions -->
       <profile>
          <id>weld3</id>
          <activation>
             <property>
-               <name>!weld2</name>
+               <name>!weldX</name>
```

Comment:
Why not just have it active by default? This looks strange.

Reply:
Profiles activated by default are disabled once `-P` is used. In fact, this profile is not needed anymore, but I'd like to keep it - could be useful once a support of a new version is added, e.g. Weld 3.1.

Reply:
Didn't realize it would be disabled... hmm, then it's probably ok